pkgs/applications/video/lbry/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, dpkg
+, fetchurl
+, at-spi2-core
+, atomEnv
+, ffmpeg
+, gtk3
+, libdrm
+, libpulseaudio
+, libuuid
+, makeWrapper
+, mesa
+, xdg_utils
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lbry-desktop";
+  version = "0.48.2";
+
+  src = fetchurl {
+    url = "https://github.com/lbryio/lbry-desktop/releases/download/v${version}/LBRY_${version}.deb";
+    sha256 = "0kqyiv9lfck9zzhiqpbvmlbyk09hajcaa2bpk1i3w03yk548an6z";
+  };
+
+  rpath = stdenv.lib.makeLibraryPath [
+    at-spi2-core
+    libdrm
+    libpulseaudio
+    libuuid
+    mesa
+    xorg.libXScrnSaver
+    xorg.libxcb
+  ] + ":${atomEnv.libPath}";
+
+  buildInputs = [
+    ffmpeg
+    gtk3
+  ];
+
+  nativeBuildInputs = [ dpkg makeWrapper ];
+
+  dontBuild = true;
+  dontPatchELF = true;
+
+  unpackPhase = ''
+      dpkg --fsys-tarfile $src | tar --extract
+      rm -rf usr/share/lintian
+  '';
+
+  installPhase = ''
```

Comment:
`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Reply:
Ah, that makes sense.